src/social/SocialIcon.tsx

```diff
@@ -160,7 +160,6 @@ const SocialIcon: React.FunctionComponent<SocialIconProps> = (props) => {
     >
       <View style={styles.wrapper}>
         <Icon
-          //@ts-ignore
```

Comment:
@flyingcircle It showing error to me

Reply:
Try installing deps again. -`yarn install`